### PR TITLE
Add App FAQs

### PR DESCRIPTION
**Pull request by gpesquero**
PR that adds a couple of FAQs in the App category, which are representative for first testing as these FAQs...

- Belong to a new category.
- One of them applies for both iOS and Android.
- The other one only applies for Android.

### CODE REVIEW

**Comment by gpesquero** [comment]
This PR also includes a translation to Spanish of one of the FAQs, but Zola fails with the error message:

`Error: Failed to build the site`
`Error: Page '/devel/organicmaps.github.io/content/faq/app/how-to-use-android-auto/index.es.md' has taxonomy 'faq' which is not defined in config.toml`

Any help to get this translation file working is appreciated!!

**Comment by gpesquero** [comment]
Commit amended to support both Spanish and English languages...

faq_app_es

**Comment by biodranik** [comment]
@gpesquero I've given you write access to the repo. Can you please push your branch here, so we can preview/test your changes?

**Comment by gpesquero** [comment]
> I've given you write access to the repo. Can you please push your branch here, so we can preview/test your changes?

@biodranik: I've pushed again my branch here (with an additional minor fix in the `config.toml` file). Is there anything else that I should do?

File: content/faq/app/which-ios-and-android-versions-are-supported/index.es.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: ¿Qué versiones de iOS y Android están soportadas?
+description: "Preguntas frecuentes de la aplicación Organic Maps"
```
Comment (biodranik) [inline content/faq/app/which-ios-and-android-versions-are-supported/index.es.md:3]: Don't you want to add a localized slug for better SEO? E.g.
`slug: que-versiones-de-ios-y-android-estan-soportadas`

File: content/faq/app/how-to-use-android-auto/index.es.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: ¿Cómo utilizar Android Auto?
+description: Preguntas frecuentes de la aplicación Organic Maps
```
Comment (biodranik) [inline content/faq/app/how-to-use-android-auto/index.es.md:3]: `slug: como-utilizar-android-auto` ?

File: content/faq/app/which-ios-and-android-versions-are-supported/index.es.md (hunk 1 above)
Comment (gpesquero) [reply content/faq/app/which-ios-and-android-versions-are-supported/index.es.md:3]: sluggized!!

File: content/faq/app/how-to-use-android-auto/index.es.md (hunk 2 above)
Comment (gpesquero) [reply content/faq/app/how-to-use-android-auto/index.es.md:3]: sluggized!!

**Review by biodranik — APPROVED** [review]
¡Gracias!

**Pull request merged**